### ISSUE TITLE
TypeScript: `matchOptions` missing from `BreadcrumbsRoute`

### ISSUE THREAD

**Issue by wkillerud** [icd2k3/react-router-breadcrumbs-hoc#42]
Hi!

Thanks for your work on this HoC and for including TS-support! 

I notice the docs and code open for giving `matchOptions` to each breadcrumb route. However, `matchOptions` is missing from the type definitions, so if I try to assign the correct type to the options given to `withBreadcrumbs` I get a type error.

Example: 

```typescript
const breadcrumbs: BreadcrumbsRoute[] = [
  {
    path: '/path/',
    breadcrumb: 'New',
    matchOptions: {
      exact: true,
    },
  },
  {
    path: '/path/:id?',
    breadcrumb: 'Edit',
  },
];
```

The type checker complains that `matchOptions` is not defined. A workaround is to remove the `BreadcrumbsRoute[]` typing, but it would be nice to avoid that.

I can open up a PR that adds the missing definitions if you'd like.

**Comment by icd2k3** [comment]
Thanks for reporting this @wkillerud!

I should be able to make & publish this change in the next day or 2 (if you beat me to it I'll be happy to merge a PR, though!).

**Comment by icd2k3** [comment]
@wkillerud this PR should fix the issue I think https://github.com/icd2k3/react-router-breadcrumbs-hoc/pull/43. Mind giving it a look? I can merge/deploy it in the near future if it looks good.

**Comment by icd2k3** [comment]
This change is now published as `2.1.5` - let me know if you still have issues with TS! (side note: looks like I have some dependency upgrades to do on the typescript end... another day 😄 )
